static/css/devhub/base.less

```diff
@@ -0,0 +1,112 @@
+@import '../common/normalize.css';
+@import '../impala/fonts/fira/fira.less';
+
+
+// Colors
+@primary-font: 'Fira Sans', sans-serif;
+@primary-font-color: #fff;
+
+@button-border: #979797;
+@primary-button-background: #29C978;
+@primary-section-background: #0568BA;
+
+
+// Common styles
+section {
+  padding: 0 20px 60px 20px;
+
+  & > h1 {
```

Comment:
Since there should only be one h1 per page, how much of this needs to be under "section"?